specifyweb/context/views.py

```diff
@@ -570,14 +570,34 @@ def views(request):
 @login_maybe_required
 @cache_control(max_age=86400, private=True)
 def viewsets(request):
-    """Retrive a list of Specify 6 viewset xml files."""
-    viewsets = []
-    for root, dir, files in os.walk(settings.SPECIFY_CONFIG_DIR):
-        for file in files:
-            if file.endswith('.views.xml'):
-                viewsets.append(
-                    os.path.relpath(os.path.join(root, file),settings.SPECIFY_CONFIG_DIR)
-                )
+    """
+    Retrieve a list of Specify 6 viewset xml files.
+    Filter out viewsets that are not applicable to the user type.
+    Filter out viewsets that are in FORM_RESOURCE_EXCLUDED_LST.
+    """
+    # Get a list of the acceptable viewsets for the user type.
+    FORM_USER_HIERARCHY = ['Admin', 'Manager', 'FullAccess', 'LimitedAccess', 'Guest']
+    USER_TYPE_DIR_NAME_MAP = {'Admin': 'admin', 'Manager': 'manager', 'FullAccess': 'fullaccess', 
```

Comment:
(optional) I like your idea of using `userType.lower()` instead of mapping dictionary